pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/NotificationService.java

```diff
@@ -125,6 +125,9 @@ public class NotificationService {
 	@Value("${preregistration.identity.name}")
 	private String fullName;
 
+	@Value("${preregistration.identity.name}")
+	private String defaultFullName;
```

Comment:
Please remove this and update the above existing config name to defaultFullName

Reply:
Removed mam and changed the existing one to defaultFullName